hack/package-system.mk

```
@@ -0,0 +1,41 @@
+.DEFAULT_GOAL=help
+.PHONY=help show diff apply delete update image
+
+help: ## Show this help.
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {sub("\\\\n",sprintf("\n%22c"," "), $$2);printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+
+show: ## Show output of rendered templates
+	case "$(NAME)" in
+    		cilium|kubeovn)
+        		kubectl get hr -n $(NAMESPACE) $(NAME) -o jsonpath='{.spec.values}' | helm template --dry-run=server -n $(NAMESPACE) $(NAME) . -f -
+        		;;
+    		*)
+        		helm template --dry-run=server -n $(NAMESPACE) $(NAME) .
+        		;;
+	esac
```

Let's keep the single command for everyhing

```suggestion
	kubectl get hr -n $(NAMESPACE) $(NAME) -o jsonpath='{.spec.values}' | helm template --dry-run=server -n $(NAMESPACE) $(NAME) . -f -
```